src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -46,9 +46,13 @@ ISO8601 Date-Times
    For brevity we may omit seconds (and minutes) from the time i.e:
    ``20000101T0000`` (``20000101T00``).
 
+   For reability we can add hyphens (``-``) between the date components
+   and colons (``:``) between the time components.
+   This is optional, but if you do this you must use both hyphens *and* colons.
```

Comment:
I'm guessing that you should have removed lines 53-54?

Reply:
No, if a point is worth making its worth making twice :rofl:

Reply:
Fixed.